examples/interop/cpp/BUILD

```diff
@@ -0,0 +1,12 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# TODO: Uncomment when macos can find cstdio.
+# load("//bazel/carbon_rules:defs.bzl", "carbon_binary")
+
+# TODO: Uncomment when macos can find cstdio.
+# carbon_binary(
+#     name = "hello_world",
+#     srcs = ["hello_world.carbon"],
+# )
```

Comment:
Can you try uncommenting this and adding `tags = ["manual"]`, to see if that's enough to get the pre-merge testing to skip this? This would be a lot more useful if we could still build this example explicitly.

Reply:
`carbon_binary` doesn't take `tags` as an argument.
I believe it currently only supports `name` and `srcs`.

Reply:
Yeah, we'll need to add `tags` support to `carbon_binary`. I've done that in #5967 and it seems to work.

Reply:
Done.